overlord/state/export_test.go

```diff
@@ -72,3 +72,15 @@ var (
 func (s *State) NumNotices() int {
 	return len(s.notices)
 }
+
+func MockGetTimeNow(f func() time.Time) (restore func()) {
```

Comment:
I think this can be removed

Reply:
I tried, and changed the calls to `MockTimeNow()`, but it doesn't work. There's no mock function to mock timenow in the `state` package. Anyway, I renamed it to MockTimeNow for consistency.

Reply:
I believe that in the `state` package it is called `MockTime`